stubs/cx.h

```diff
@@ -0,0 +1,122 @@
+#ifndef STUBS_CX_H
+#define STUBS_CX_H
+
+#include "mock_engine.h"
+#include "ox.h"
+
+// cx flags
+#define CX_FLAG
+#define CX_LAST        (1 << 0)
+#define CX_RND_TRNG    (2 << 9)
+#define CX_RND_RFC6979 (3 << 9)
+
+#define CX_SHA256_SIZE 32
+
+typedef uint32_t cx_err_t;
+
+// keypairs
+typedef struct {
+    // cx_curve_t curve;
+    size_t W_len;
+    uint8_t W[65];
+} cx_ecfp_public_key_t;
+typedef struct {
+    // cx_curve_t curve;
+    size_t d_len;
+    uint8_t d[32];
+} cx_ecfp_private_key_t;
+
+/** Message Digest algorithm identifiers. */
+typedef enum {
+    CX_NONE = 0,
+    CX_RIPEMD160 = 1,  // xx bytes
+    CX_SHA256 = 3,     // 32 bytes
+} cx_md_t;
+
+typedef struct {
+    // struct cx_hash_header_s header;
+    size_t blen;
+    uint8_t block[64];
+    uint8_t acc[8 * 4];
+} cx_sha256_t;
+
+typedef struct cx_ripemd160_s {
+    // struct cx_hash_header_s header;
+    size_t blen;
+    uint8_t block[64];
+    uint8_t acc[5 * 4];
+} cx_ripemd160_t;
+
+// MOCK: cx_hash_t
+typedef struct {
+    cx_md_t md_type;
+    uint8_t buf[32];
+} cx_hash_t;
+
+// Missing:
+//
+// os_perso_derive_node_bip32
```

Comment:
It's in the os.h file, right? So not missing

Reply:
This was a list of what I needed to add, forgot to cleanup